.github/ISSUE_TEMPLATE/new-team-member.md

```diff
@@ -22,7 +22,7 @@ Follow the checklist below to onboard a new team member!
 **Communication**
 
 - [ ] Send them a link to the [2i2c Team Compass](https://team-compass.2i2c.org/en/latest/) and its [Team Operations section](https://team-compass.2i2c.org/en/latest/operations/index.html)
-- [ ] Send them a link to the [2i2c Google Drive](https://drive.google.com/drive/folders/1ABxxSFycGfCzQc9czfwer_dat-GVi4jw?usp=sharing)
+- [ ] Send them a link to the [2i2c Team Google Drive](https://drive.google.com/drive/u/1/folders/0AJcabtB-T0LnUk9PVA)
```

Comment:
Are people granted access to the shared drive just by being part of the 2i2c.org google organization, or because someone is inviting them to it?

Reply:
I figure no matter what, getting a link to it so they are aware of it is a good thing - and as part of that, they will learn if they need to be granted additional permissions etc anyhow I guess.

Let's go with this for now!

Reply:
I believe we get access just by being part of the 2i2c.org google organization. But is good to double check and as you said, it's worth signaling that it exists.
 
 Thanks for the merge @consideRatio 🎉